### DISCUSSION TITLE
No module named 'spacy.gold' while loading model using spacy.load("en_core_web_lg")

### DISCUSSION THREAD

**Discussion by aniruddhakal** [explosion/spaCy#9320] [Help: Installation] [answered]
## How to reproduce the behaviour

`import spacy`
`nlp = spacy.load("en_core_web_lg")`

Throws the following error:
`ModuleNotFoundError: No module named 'spacy.gold'`

## Your Environment

* Operating System: 20.0.4
* Python Version Used: python3.8
* spaCy Version Used: 3.1.3

What's the new way to load "en_core_web_lg" in spacy 3.1.3?

**Comment by polm** [comment, marked as answer]
That code should still work. It sounds like you upgraded spaCy from v2 but didn't re-download the model? You have to update your models too, as they've been updated for 3.0.
